Makefile

```diff
@@ -21,7 +21,9 @@ certificates/domain.key:
 
 .PHONY: .create-secrets
 .create-secrets:
-	@$(MAKE_C) certificates deploy
+	@echo "Creating docker secrets..." && \
```

Comment:
minor: ```#Creating docker secrets...``` putting it as a comment does the same as your echo here. it makes it slightly more readable.

```bash
# Creating docker secrets...
@$(MAKE_C) certificates deploy
# Done: Creating docker secrets
```

Reply:
thanks! will address